docs/user-guide/builtins.md

```diff
@@ -133,23 +133,32 @@ should also be defined as `async`.
 from pyscript import fetch
 
 
-response = await fetch("https://example.com").text()
+response = await fetch("https://example.com")
+if response.ok:
+    data = await response.text()
+else:
+    print(response.status)
 ```
 
-If the `fetch` operation _causes a response that is not deemed `OK`_ (the
-definition of which
-[can be found here](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok))
-then an exception is raised.
+Alternatively, rather than using a double `await` (one to get the response, the
+other to grab the data), it's possible to chain the calls into a single
+`await` like this:
 
-Assuming an `OK` response, the following methods are available to you to access
-the data returned from the server:
+```python title="A simple HTTP GET as a single `await`"
+from pyscript import fetch
+
+data = await fetch("https://example.com").text()
```

Comment:
something odd with the highlighter ... but also something odd with the initial `python` thing as title is probably not understood in here ... if that's expected, feel free to ignore this comment!

Reply:
to clarify, that's what GitHub produces and I think if we have a repo with `.md` files, those files should be compliant with what GitHub UI offers ... was that just a typo in the file? 🤔

Reply:
Agreed. Changing now.